src/config/GameConfig.java

```diff
@@ -1,11 +1,30 @@
 package config;
 
+import de.ur.mi.oop.colors.Color;
+import de.ur.mi.oop.colors.Colors;
```

Comment:
Is there a difference between: de.ur.mi.oop.colors.Color &
                                                de.ur.mi.oop.colors.ColorS

Reply:
`Color` represents a single Color. `Colors` is a utility class with a list of predefined Colors (i.g. `Colors.WHITE`).